src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
The main thing I want here is for when new versions of OSX come out, .NET apps aren't immediately broken by default just because the RID changed.  If Apple decides to make breaking changes, so be it, we can't help that.  But we can build our product in such a way that if there are no operating system breaking changes from osx.10.12 => osx.10.13, then .NET apps will just work.

[WWDC17](https://developer.apple.com/wwdc/) takes place June 5-9.  It would be bad if we shipped just before that and we were immediately broken by a new OS that was announced there.

Setting `DOTNET_RUNTIME_ID` is a total hack, we shouldn't use it as a crutch.  I'm glad the hack is there to get me unblocked, but it shouldn't be our default answer.